zeppelin-web/src/app/helium/helium.controller.js

```diff
@@ -149,37 +149,55 @@ export default function HeliumCtrl($scope, $rootScope, $sce,
     return license;
   }
 
-  $scope.enable = function(name, artifact, type, groupId) {
+  const getHeliumTypeText = function(type) {
+    if (type === HeliumType.VISUALIZATION) {
+      return `<a href="https://zeppelin.apache.org/docs/latest/development/writingzeppelinvisualization.html">${type}</a>`; // eslint-disable-line max-len
```

Comment:
I will add `target="_blank"` 
Regarding to `zeppelin_version`, it make us to modify this file whenever we changes version. 😭 ..